src/model/kaniRunner.ts

```diff
@@ -258,7 +268,7 @@ function sendErrorToChannel(output: CommandOutput, args: string[]): void {
 
 // Creates a unique name and adds a channel for the harness output to Output Logs
 function sendOutputToChannel(output: CommandOutput, args: string[]): void {
-	const harnessName = args.at(1)!;
+	const harnessName = args.at(args.length - 1)!;
```

Comment:
Places like this I'd add [invariants](https://www.npmjs.com/package/ts-invariant) (e.g., `args.length != 0`).

Reply:
Can do! Thanks for pointing it out